app/controllers/check_your_answers_controller.rb

```diff
@@ -2,5 +2,6 @@ class CheckYourAnswersController < ApplicationController
   def show
     @personal_details = PersonalDetails.last
     @contact_details = ContactDetails.last
+    @degree_details = Degree.last
```

Comment:
`degree_details` feels awkward here - in fact `_details` on the back of all of these models feels like it's just adding noise rather than understanding.

Don't need to address this now, just thinking out loud about these model names as we flesh out the service.

Reply:
I agree, in this case, `degree` is a thing and it does not behave like an adjective,  as in the case of `Personal`.

Reply:
If you accept this I'll make the suggested improvements in the following PR (I suppose it is ok)

Reply:
Sure, happy for us to address this later.